docs/manual/src/kotlin/gradle.md

```diff
@@ -42,3 +42,18 @@ allprojects {
     }
 }
 ```
+
+## JNA dependency
+
+UniFFI relies on [JNA] for the ability to call native methods.
+JNA 5.7 or greater is required.
```

Comment:
Could you please also add an entry to the changelog to advertise this requirement? It feels reasonable to me to call this out as a breaking change, since if a consumer updates UniFFI without updating a pinned version of JNA, they might start getting crashes.